openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIParameterMapper.java

```diff
@@ -108,7 +108,14 @@ public void getResourceInputs(Components components, SemanticModel semanticModel
                     QualifiedNameReferenceNode referenceNode =
                             (QualifiedNameReferenceNode) requiredParameterNode.typeName();
                     String typeName = (referenceNode).modulePrefix().text() + ":" + (referenceNode).identifier().text();
-                    if (typeName.equals(HTTP_REQUEST)) {
+                    //TODO: add warning common hhtp:Resuqt not caontain details in get method
+                    if (typeName.equals(HTTP_REQUEST) && (Constants.GET.toLowerCase(Locale.ENGLISH).equalsIgnoreCase(
```

Comment:
Do we need to use `toLowerCase()` in here since we are using `equalsIgnoreCase()` to compare?

Reply:
Yeah we don't need it, by mistakenly adding both, will remove toLowerCase() .